src/typography.scss

```diff
@@ -0,0 +1,40 @@
+:root {
+  // Font sizes
+  --font-size-1: 14px;
+  --font-size-2: 14px;
+  --font-size-3: 16px;
+  --font-size-4: 18px;
+  --font-size-5: 24px;
+  --font-size-6: 32px;
+
+  // Line heights
+  --font-line-height-1: 18px;
+  --font-line-height-2: 24px;
+  --font-line-height-3: 24px;
+  --font-line-height-4: 24px;
+  --font-line-height-5: 32px;
+  --font-line-height-6: 52px;
+
+  // Font weights
+  --font-weight-very-light: 200;
+  --font-weight-light: 300;
+  --font-weight-normal: 400;
+  --font-weight-bold: 500;
+
+  // Font family
+  --font-family: "Roboto", "Open Sans", "Noto Kufi Arabic", "Noto Sans JP", sans-serif;
+
+  // Fonts
+  --font-h1: var(--font-weight-bold) var(--font-size-6)/var(--font-line-height-6) var(--font-family);
+  --font-h2: var(--font-weight-bold) var(--font-size-5)/var(--font-line-height-5) var(--font-family);
+  --font-h3: var(--font-weight-bold) var(--font-size-5)/var(--font-line-height-5) var(--font-family);
+  --font-h4: var(--font-weight-bold) var(--font-size-4)/var(--font-line-height-4) var(--font-family);
+  --font-h5: var(--font-weight-bold) var(--font-size-3)/var(--font-line-height-3) var(--font-family);
+  --font-general-label: var(--font-weight-normal) var(--font-size-2)/var(--font-line-height-2) var(--font-family);
+  --font-paragraph: var(--font-weight-normal) var(--font-size-3)/var(--font-line-height-3) var(--font-family);
+  --font-subtext: var(--font-weight-normal) var(--font-size-1)/var(--font-line-height-1) var(--font-family);
+
+  // Font smoothing
+  --webkit-font-smoothing: antialiased;
```

Comment:
don't use the webkit prefix, it is confusing (too similar) to the real vendor prefix

Reply:
what to do instead? put it in the end of the variable?